lib/indy/README.md

```diff
@@ -0,0 +1,175 @@
+
+## Solution Walkthrough
+
+### Setup Cloud9
+
+We will use AWS Cloud9 to execute the subsequent commands. Follow the instructions in [Cloud9 Setup](../../docs/setup-cloud9.md)
+
+### Clone this repository and install dependencies
+
+```bash
+git clone https://github.com/aws-samples/aws-blockchain-node-runners.git
+cd aws-blockchain-node-runners
+npm install
+```
+
+**NOTE:** In this tutorial we will set all major configuration through environment variables, but you also can modify parameters in `config/config.ts`.
+
+### Deploy Indy Nodes
+
+Indy Network is built using 4 EC2 instances for Steward and 1 EC2 instance for Trustee. Various information such as DID is acquired in the following procedure, copied by referring to [this community spreadsheet](https://docs.google.com/spreadsheets/d/1LDduIeZp7pansd9deXeVSqGgdf0VdAHNMc7xYli3QAY/edit#gid=0).
+
+#### Building resources
+
+1. Install npm dependency packages
+
+```bash
+cd lib/indy
+pwd
+# Make sure you are in aws-blockchain-node-runners/lib/indy
+npm install
+```
+
+2. Setting up initial AWS Cloud Development Kit (CDK)
+
+The following command is executed only when using AWS CDK for the first time in the region where the deployment will be carried out.
+
+```bash
+npx cdk bootstrap
+```
+
+3. Deploying resources with CDK
+
+```bash
+npx cdk deploy
+
+Outputs:
+IndyNodeStack.Node1InstanceId = i-xxxxxxxxxxxxxxxxx
+IndyNodeStack.Node2InstanceId = i-xxxxxxxxxxxxxxxxx
+IndyNodeStack.Node3InstanceId = i-xxxxxxxxxxxxxxxxx
+IndyNodeStack.Node4InstanceId = i-xxxxxxxxxxxxxxxxx
+IndyNodeStack.TrusteeInstanceId = i-xxxxxxxxxxxxxxxxx
+```
+
+**NOTE:** User data for the Steward instance is created by referring to [the Community Docs](https://github.com/hyperledger/indy-node/blob/main/docs/source/install-docs/AWS-NodeInstall-20.04.md).
+
+#### Setting up Trustee
+
+Log in to the Trustee instance via Session Manager from the EC2 (or Systems Manager) console and generate Trustee/Steward DIDs.
+​
+
+```bash
+cd /
+./indy-cli-rs
+​
+# Perform the following commands 3 times for Trustee and 4 times for Steward
+wallet create <WALLET_NAME> key=<KEY>
+wallet open <WALLET_NAME> key=<KEY>
+did new seed=<SEED>
+wallet close
+```
+
+#### Setting up Steward
+
+Log in to the Steward instance via Session Manager from the EC2 (or Systems Manager) console and generate Validator verkey, BLS key, and BLS POP.
+
+```bash
+sudo init_indy_node <ALIAS> <NODE_IP> 9701 <CLIENT_IP> 9702 <SEED>
+```
+
+**NOTE:** Here, Steward represents Validator Node ([reference information](https://github.com/pSchlarb/indy-node/blob/documentationUpdate/docs/source/installation-and-configuration.md#32-validator-node-installation)).
+
+#### Generating Genesis files
+
+1. Download each sheet (Stewards, Trustees) containing the information generated in the steps so far
```

Comment:
Need guidance where I should go to download the files from? I've got stuck on this step and could not continue.